scripts/docker/readme.md

```diff
@@ -40,8 +41,14 @@ and --focal is used for ubuntu 16.04, 18.04, and 20.04 docker images, respective
 To run the docker container:
 
     ./run.sh
+
 It will automatically detect the current Ubuntu OS version. If a specific version is desired, the option
 --xenial, --bionic, and --focal is used for ubuntu 16.04, 18.04, and 20.04 docker images, respectively.
+To add arguments to the launch file besides `dds:=false robot:=sim_pub` you can do instead:
```

Comment:
Can you be a bit more clear that the args adds arguments *in addition to* these, it doesn't replace them.